lib/order.rb

```diff
@@ -0,0 +1,15 @@
+class Order
+  attr_reader :dish, :price, :order, :quantity
```

Comment:
It doesn't make sense for `dish` and `quantity` to be attributes as an order is a collection of these.

`order` should not be exposed as an attribute.  This class is the order, so it doesn't make sense for it to have an order property anyway.